crates/polars-time/src/round.rs

```diff
@@ -44,3 +44,36 @@ impl PolarsRound for DateChunked {
             .into_date())
     }
 }
+
+#[cfg(feature = "dtype-duration")]
+impl PolarsRound for DurationChunked {
+    fn round(&self, every: Duration, offset: Duration, _tz: Option<&Tz>) -> PolarsResult<Self> {
+        if every.negative {
+            polars_bail!(ComputeError: "cannot round a Duration to a negative duration")
+        }
+        if !every.is_constant_duration() {
+            polars_bail!(InvalidOperation: "Cannot round a Duration series to a non-constant duration.");
+        }
+        if !offset.is_zero() {
+            polars_bail!(InvalidOperation: "Offsets are not supported for rounding Durations.");
+        }
```

Comment:
could you use polars_ensure here, and pass the time zone to `is_constant_duration`? See

https://github.com/pola-rs/polars/blob/0e68e7535870dc921653abc535d2f9afe5078f37/crates/polars-plan/src/dsl/function_expr/ewm_by.rs#L12-L19

for an example of how to do it

Reply:
I've passed in the time zone, but am I handling the timezone feature flag correctly?

I'm not sure that the timezone will ever have a value here, since durations don't have timezones. I only saw `round` called for a duration once, and the timezone argument was a hard-coded `None`. `round` only takes in a timezone so that the trait can support datetimes. A simpler, if brittle, alternative would be to just ensure the timezone is `None`.

Also, want to flag that the time-zone-aware `is_constant_duration` now counts days and weeks to be constant frequencies when the timezone is none. I've updated the tests to match.

Reply:
> A simpler, if brittle, alternative would be to just ensure the timezone is None.

yup, this would be fine, thanks!